File: OOP/pattern.c
#include <stdio.h>

void pattern(char ch, int len) {
  if (len) {
    pattern(ch, len - 1);
  }
  for (int i = 0; i < len; i++) {
    printf("%c ", ch);
  }
  printf("\n");
}

int main() {

  pattern('K', 5);
  return 0;
}

File: Java/school/Pattern.java
class Pattern
{
    void main()
    {
        int NOL = 5; 
        for(int line = 1 ; line <= 5 ; line++)
        {
            for(int j = 1 ; j <= (NOL - line) ; j++)
            System.out.print("*");
            System.out.println(line);
        }
    }
}